containers/ecr-viewer/src/app/tests/date-utils.test.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  DateRangeOptions,
+  convertDateOptionToDateRange,
+} from "@/app/view-data/utils/date-utils";
+
+describe("convertDateOptionToDateRange", () => {
+  const today = new Date("2024-04-03");
```

Comment:
```suggestion
  const today = new Date("2024-04-03T12:34");
```
to make sure the zeroing works properly (not sure if this date is fully valid, so might need tweaking)

Reply:
Just noting for review, ended up changing to `2024-04-02T12:34` instead of `04-03` to effectively keep the same date (before, 2024-04-03 midnight UTC was 04-02 8pm in Jest local TZ).